README.md

```diff
@@ -121,6 +122,41 @@ Example: `replica.postgres-cluster.service.dc1.consul`, `replica.postgres-cluste
 
 It requires the installation of a consul in client mode on each application server for service DNS resolution (or use [forward DNS](https://developer.hashicorp.com/consul/tutorials/networking/dns-forwarding?utm_source=docs) to the remote consul server instead of installing a local consul client).
 
+
+### [Type D] PostgreSQL High-Availability with HAProxy Load Balancing and Patroni raft
```

Comment:
These are all the same type A and type B schemes, just instead of etcd, you can choose  "patroni_raft" in dcs_type

Reply:
consider just adding a comment about dcs_type: "patroni_raft" to the type A schema